### PR TITLE
feat: Update const.py for Dual Doorbells #303

### PR DESCRIPTION
**Pull request by ejpenney**
This ... sort of works in my environment with a wired doorbell dual (#303).   I'm finding camera streams to be VERY low quality, image gets garbled real quickly and is un-usable about 25% of the time.  Also, sensor.doorbell_detected_person_name is never getting set.  Not sure if these are related to this device specifically or a separate bug.

### CODE REVIEW

File: custom_components/eufy_security/const.py [hunk 1]
```diff
@@ -288,6 +288,8 @@ class DEVICE_TYPE(Enum):
     SOLO_CAMERA_SPOTLIGHT_1080 = 60
     SOLO_CAMERA_SPOTLIGHT_2K = 61
     SOLO_CAMERA_SPOTLIGHT_SOLAR = 62
+    BATTERY_DOORBELL_DUAL = 91
```
Comment (TomK) [inline custom_components/eufy_security/const.py:291]: I'm not sure this constant name is correct. My doorbell is a wired dual:
```
eufy_security - Exception - process_messages: 91 is not a valid DEVICE_TYPE
```

Comment (TomK) [reply custom_components/eufy_security/const.py:291]: Maybe `DOORBELL_DUAL_8213` (suffix model number) would be interesting to see the model number of the 93 device

Comment (TomK) [reply custom_components/eufy_security/const.py:291]: For more context, my 91 device supports both wired or battery

Comment (ejpenney) [reply custom_components/eufy_security/const.py:291]: You're right, I had a momentary lapse in sanity and swapped wired and battery device IDs. Let me swap them back.

Comment (ejpenney) [reply custom_components/eufy_security/const.py]: Fixed.  Yeah, it "works" either way, but in the interest of correctness and future-proofing it'd be better to have the device types correctly labeled!

**Comment by TomK** [comment]
@fuatakgun this PR can probably get merged, and the lack of streamed/live video could be addressed in another PR, with a more specific issue being created.

**Comment by fuatakgun** [comment]
Done thanks a lot

**Pull request merged**